ampersand-filtered-subcollection.js

```diff
@@ -157,10 +157,15 @@ assign(FilteredCollection.prototype, Events, {
         this._watched = difference(this._watched, isArray(item) ? item : [item]);
     },
 
-    _sortModels: function (newModels) {
-        var comparator = this.comparator || this.collection.comparator;
+    _sortModels: function (newModels, comparator) {
+        comparator = comparator || this.comparator || this.collection.comparator;
         if (comparator) {
-            newModels = sortBy(newModels, comparator);
+            if (typeof comparator === 'string' || comparator.length === 1) {
+                newModels = sortBy(newModels, comparator);
```

Comment:
@kamilogorek @lukekarrys is correct. `sortBy` is still used when the comparator is a string or a function with 1 argument